jsonpb/jsonpb.go

```diff
@@ -116,13 +127,43 @@ type wkt interface {
 	XXX_WellKnownType() string
 }
 
+// customJSON can be implemented by messages that have custom non-object JSON
+// representations. If the message implements this method and the method
+// returns true, the JSON representation will not be an object (for example,
+// the message marshals itself as a string).
+type customJSON interface {
+	XXX_CustomJSON() bool
```

Comment:
I did see your question a few moments ago.

Sure, I can cut this stuff out for now. It's not terribly complicated to add back later if/when a more concrete need arises.